### DISCUSSION TITLE
How can we manipulate/work with response data?

### DISCUSSION THREAD

**Discussion by mattnield** [postmanlabs/postman-flows#306] [Q&A] [answered]
How can we go about working with the response data from a request? 

Here's a use case: Using the [Open Breweries DB API](https://www.openbrewerydb.org/), I want to get a list of all of the breweries. I want to pass in the name and phone numbers of the breweries to the body of another API call. Just those details though.

So from this:
```json
[
    {
        "id": "banjo-brewing-fayetteville",
        "name": "Banjo Brewing",
        "brewery_type": "planning",
        "street": null,
        "address_2": null,
        "address_3": null,
        "city": "Fayetteville",
        "state": "West Virginia",
        "county_province": null,
        "postal_code": "25840",
        "country": "United States",
        "longitude": null,
        "latitude": null,
        "phone": "3042164231",
        "website_url": null,
        "updated_at": "2021-10-23T02:24:55.243Z",
        "created_at": "2021-10-23T02:24:55.243Z"
    },
    ....
    {
        "id": "dirt-road-brewing-corvallis",
        "name": "Dirt Road Brewing",
        "brewery_type": "planning",
        "street": null,
        "address_2": null,
        "address_3": null,
        "city": "Corvallis",
        "state": "Oregon",
        "county_province": null,
        "postal_code": "97330-3005",
        "country": "United States",
        "longitude": null,
        "latitude": null,
        "phone": "5415984221",
        "website_url": "http://dirtroadbrewing.com",
        "updated_at": "2021-10-23T02:24:55.243Z",
        "created_at": "2021-10-23T02:24:55.243Z"
    }
]
```
I want to end up with this:
```json
[
    {
        "name": "Banjo Brewing",
        "phone": "3042164231"
    },
    ....
    {
        "name": "Dirt Road Brewing",
        "phone": "5415984221"
    }
]
```

Is this something we should be able to do? I've had a try at using the expression builder in the Durables block, but perhaps this isn't the right way to go about it?

**Comment by samay-v** [comment, marked as answer]
Hey @mattnield 
I built a flow for you to demonstrate how you can achieve it, let me know if that solves your use case 😄 
https://www.postman.com/postman/workspace/postman-answers/flow/626a7df1d91bbd014d59aedd

**Reply by mattnield** [reply]
Thank you @samay-v 
I hadn't thought about using the loop to aggregate the individual rows together!

**Reply by mattnield** [reply]
I also just noticed that the web version contains many more shapes!

**Reply by samay-v** [reply]
The app and the web versions have the same amount of blocks, maybe try updating your app?

**Comment by mattnield** [comment]
👍 turns out Id disabled automatic updates for some reason 😖
